dso-l2/pom.xml

```diff
@@ -131,6 +131,11 @@
       <groupId>commons-cli</groupId>
       <artifactId>commons-cli</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.glassfish.jaxb</groupId>
+      <artifactId>jaxb-runtime</artifactId>
+      <version>2.3.1</version>
+    </dependency>
   </dependencies>
```

Comment:
we cannot add this.  Can the code requiring this be removed or modified?

Reply:
All the tests integration-tests within galvan-support module starts failing without this dependency
 java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException
[ClientLeakIT][testServer0] 	at com.tc.config.DefaultConfigurationProvider.getTcConfiguration(DefaultConfigurationProvider.java:140)
[ClientLeakIT][testServer0] 	at com.tc.config.DefaultConfigurationProvider.initialize(DefaultConfigurationProvider.java:106)
[ClientLeakIT][testServer0] 	at com.tc.server.TCServerMain.main(TCServerMain.java:66)
[ClientLeakIT][testServer0] Caused by: java.lang.ClassNotFoundException: javax.xml.bind.JAXBException
[ClientLeakIT][testServer0] 	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:583)
[ClientLeakIT][testServer0] 	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
[ClientLeakIT][testServer0] 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
[ClientLeakIT][testServer0] 	... 3 more

Reply:
There are some tests failing in galvan module because of java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException when running with java11 at the time of startup of server.

Reply:
I think that change belongs in tc-configuration, not in dso-l2.  There should be no jab requirements tc-core.

Reply:
Removed this dependency we don't need it